src/main/service/Operation/Impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package main.service.Operation.Impl;
+
+import main.service.Operation.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public int perform(int balance, int quantity) {
+        return balance;
+    }
```

Comment:
This class does nothing. 
Why do we need it?

Here should be some logic.
```suggestion
public class BalanceOperationHandler implements OperationHandler {
    @Override
    public int perform(int balance, int quantity) {
        return balance;
    }
```